src/Modules/StatCan.OrchardCore.ContentFields/Views/MultivalueFieldEditorSettings.Edit.cshtml

```diff
@@ -0,0 +1,111 @@
+@using StatCan.OrchardCore.ContentFields.Multivalue.Settings
+@model StatCan.OrchardCore.ContentFields.Multivalue.ViewModels.MultivalueFieldSettingsViewModel
+
+<script asp-src="~/StatCan.OrchardCore.ContentFields/Scripts/optionsEditor.min.js"
+  debug-src="~/StatCan.OrchardCore.ContentFields/Scripts/optionsEditor.js" asp-name="optionsEditor" at="Foot"
```

Comment:
you might want to modify the script name to something else here. Unless you are using a unmodified version of Orchard's optionsEditor script.

If you are using a unmodified version, do not include this script in the wwroot and simply reference oc's script. If you are using a modified version, change the name.